src/Server/OperationParams.php

```diff
@@ -84,57 +106,69 @@ public static function create(array $params, bool $readonly = false): OperationP
             'extensions' => null,
         ];
 
-        if ($params['variables'] === '') {
-            $params['variables'] = null;
-        }
-
-        // Some parameters could be provided as serialized JSON.
-        foreach (['extensions', 'variables'] as $param) {
-            if (! is_string($params[$param])) {
-                continue;
-            }
-
-            $tmp = json_decode($params[$param], true);
-            if (json_last_error() !== JSON_ERROR_NONE) {
-                continue;
+        array_walk($params, static function (&$value): void {
```

Comment:
array_map is more appropriate here

Reply:
I explicitly want to mutate the array, so `array_map` is not an option. I just noticed a good old `foreach` works as well though.